### PR TITLE
#21564 now the template could be created on a specific site

### LINKED ISSUE #21564: Accept host id on create template
Currently the template is stored on the current host, however the rest client should has the ability to send the host id in order to store the template on a particular host (optional value)

### PR DESCRIPTION
**Pull request by jdotcms**
now the template could be created on a specific site

### CODE REVIEW

File: dotCMS/src/curl-test/CreateAdvanceTemplateOnSpecificHost.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,231 @@
+{
+	"info": {
```
Comment (erickgonzalez) [inline dotCMS/src/curl-test/CreateAdvanceTemplateOnSpecificHost.postman_collection.json:2]: Let's move this test to the **TemplateResource collection** that already exists.

**Review by freddyucv — APPROVED** [review]
No forget test when a Template contains File Container using a relative path

File: dotCMS/src/curl-test/CreateAdvanceTemplateOnSpecificHost.postman_collection.json (hunk 1 above)
Comment (jdotcms) [reply dotCMS/src/curl-test/CreateAdvanceTemplateOnSpecificHost.postman_collection.json:2]: I prefer to keep it separated

**Comment by jdotcms** [comment]
> No forget test when a Template contains File Container using a relative path

we are just adding a new property, relative and absolute should work as usual

**Review by erickgonzalez — APPROVED** [review]

**Pull request merged**